packages/core/strapi/src/container.ts

```diff
@@ -24,7 +24,7 @@ export const createContainer = (strapi: Strapi): Container => {
         const resolver = registered.get(name);
 
         if (typeof resolver === 'function') {
-          resolved.set(name, resolver({ strapi }, args));
+          resolved.set(name, resolver(this, args));
```

Comment:
Is Strapi attached to the container after being created? What does this change mean exactly?

Reply:
We don't use this atm but the goal is to pass the container instead of strapi